docs/examples/data/torchvision/data.py

```diff
@@ -0,0 +1,54 @@
+"""Make sure the data is available"""
+import os
+import shutil
+import sys
+import time
+from multiprocessing import Pool
+from pathlib import Path
+
+from torchvision.datasets import INaturalist
+
+
+def link_file(src:str, dest:str):
+    Path(src).symlink_to(dest)
+
+
+def link_files(src:str, dest:str, workers=4):
+    src = Path(src)
+    dest = Path(dest)
```

Comment:
This is already receiving paths:
```suggestion
def link_files(src: Path, dest: Path, workers: int = 4) -> None:
```